yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -456,7 +456,11 @@ function makeRollupTx(l2Block: L2Block) {
   const input = encodeFunctionData({
     abi: RollupAbi,
     functionName: 'propose',
-    args: [{ header, archive, blockHash, txHashes: [] }, [], body],
+    args: [
+      { header, archive, blockHash, oracleInput: { provingCostModifier: 0n, feeAssetPriceModifier: 0n }, txHashes: [] },
```

Comment:
also @todo #9963?

Reply:
For an archiver test not sure we would really need to change it though 🤷.